### PR TITLE
Bump to gobrake v4

### PR DESCRIPTION
**Pull request by darigaaz**

### CODE REVIEW

**Comment by gravis** [comment]
Thanks for your contribution! 
Is there a way you can make the tests pass as well? It's probably just about updating the go test matrix on in `.travis-ci.yml`.

**Comment by darigaaz** [comment]
looks like gobrake does not work with gopkg.in anymore, complains on usage of gobrake internal package
```
import github.com/airbrake/gobrake/internal/smthing
```
trying to fix it atm

**Comment by darigaaz** [comment]
1. airbrake/gobrake does not work with gopkg.in anymore https://github.com/airbrake/gobrake/issues/53
2. instead it uses "github.com/airbrake/gobrake/v4" but this semantics requires go modules enabled
3. some internals in gobrake/v4 requires at least go 1.11 (it wont compile no matter what GO111MODULES is set)

so minimum go version is 1.11 now and go modules must be enabled, otherwise `gobrake "github.com/airbrake/gobrake/v4"` will complain

and I the easiest way to make it work is to fix build matrix and convert it to go module

I dont know if is it acceptable to convert this package to module
but I tried everything else

**Comment by gravis** [comment]
If we bump the version to v4, I think it's completely acceptable, and I wanted to do it anyway.  We can't continue to support Go versions that are not supported themselves.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # logrus-airbrake-hook Changelog
 
+v4.0.0 - 2020-01-20
+
+* Bump gobrake v4
```
Comment (gravis) [inline CHANGELOG.md:5]:
```suggestion
* Bump gobrake v4
* Add support for Go modules
```

**Review by gravis — APPROVED** [review]
LGTM!

**Comment by gravis** [comment]
Thanks a lot for your contribution

**Pull request merged**